src/App.js

```diff
@@ -117,25 +117,26 @@ function App() {
     fetchData();
   }, []);
 
-  //set a side effect hook to save the todolist in browser's local storage
-  React.useEffect(() => {
-    if (!isLoading) {
-      localStorage.setItem("savedTodoList", JSON.stringify(todoList));
-    }
-  }, ["savedTodoList", JSON.stringify(todoList)]); // with stringify we can read the values inside todolist array
-
-  // const [todoList, setTodoList] = useSemiPersistentState();
-
   return (
-    <>
-      <h1>Todo List</h1>
-      <AddTodoForm onAddTodo={addTodo} />
-      {isLoading ? (
-        <p>Loading</p>
-      ) : (
-        <TodoList todoList={todoList} onRemoveTodo={removeTodo} />
-      )}
-    </>
+    <BrowserRouter>
+      <Routes>
+        <Route
+          path="/"
+          exact=""
+          element={
+            <>
+              <h1>Todo List</h1>
+              <AddTodoForm onAddTodo={addTodo} />
+              {isLoading ? (
+                <p>Loading</p>
+              ) : (
+                <TodoList todoList={todoList} onRemoveTodo={removeTodo} />
+              )}
+            </>
+          }
+        ></Route>
```

Comment:
Great, but it looks like a new route has not been created.

Reply:
Sorry, it looks like the latest changes did not push in Github, I just push it now.